### PR TITLE
fix(commitment): fix commitment english text

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `fix/commitment-content-english`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-7479
| License          | BSD 3-Clause

## Description

Fix english language translation. Specific text has been mentioned in translation request.

## Related

TRANS-37954

### CODE REVIEW

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "billing_commitment_commit": "Subscribe and get a discount",
   "billing_commitment_commit_again": "Re-subscribe and get a discount",
-  "billing_commitment_none": "You do not have a commitment on your {{ serviceName }} subscription. You can get a discount today if you add a commitment to your &lt;dedicated server>{{ serviceType }} subscription.",
+  "billing_commitment_none": "You do not have a commitment on your {{serviceName}} subscription. You can get a discount today if you add a commitment to your {{serviceName}} {{serviceType}} subscription.",
```
Comment (marie-j) [inline packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json:4]: Shouldn't it be <serviceType> <serviceName> ?

Comment (varun257) [reply packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json:4]: They asked for exact text change. So we did not change anything else.

File: packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   "billing_commitment_payment_method_none": "You do not have a valid payment method.",
   "billing_commitment_payment_method_add": "Add a new payment method ",
   "billing_commitment_summary": "Finalizing",
-  "billing_commitment_summary_description_periodic": "Starting {{ renewDate }} your default payment method will automatically be charged [$XX.XX] every month for {{ serviceName }} until {{ engagementEndDate }}.",
+  "billing_commitment_summary_description_periodic": "Starting {{renewDate}} your default payment method will automatically be charged {{price}} every month for {{serviceName}} until {{engagementEndDate}}.",
```
Comment (antleblanc) [inline packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json:22]: Missing a translate-value, isn't it?

see: https://github.com/ovh/manager/blob/fix/commitment-content-english/packages/manager/modules/billing-components/src/components/commitment/commitment.html#L151-L153

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json (hunk 2 above)
Comment (antleblanc) [reply packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json:22]: I didn't pay attention to the actual diff, sorry.

https://github.com/ovh/manager/pull/5520/files#diff-a8173cdfcadae3ac58bfba01e641ac0d0b004c225e753909ca9e69ae4946d490R145

**Review by antleblanc — APPROVED** [review]

**Pull request merged**